.dockerignore

```diff
@@ -0,0 +1,22 @@
+.git
+.gitignore
+.dockerignore
+Dockerfile
+hokusai
+
+README.md
```

Comment:
Curious why some files are here?

Reply:
So they are excluded from the Docker image build